modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -1223,6 +1223,9 @@ public boolean hasPendingServerExchange() {
      * @return Last topology version before the provided one when affinity was modified.
      */
     public AffinityTopologyVersion lastAffinityChangedTopologyVersion(AffinityTopologyVersion topVer) {
+        if (topVer == null)
```

Comment:
I'm not sure that is a valid check. Possibly we shouldn't be there in such a case at all. Let's ask @sergey-chugunov-1985